app/src/sharedTest/java/org/oppia/android/app/parser/HtmlParserTest.kt

```diff
@@ -162,7 +162,7 @@ class HtmlParserTest {
   }
 
   @Test
-  fun testHtmlContent_customSpan_isAdded() {
+  fun testHtmlContent_customSpan_isAddedWithCorrectlySpacedLeadingMargin() {
```

Comment:
I missed this earlier, but in my original comment I actually meant for you to create a new test suite to house this test (e.g. a CustomBulletSpanTest). Can you create a new test suite + test to verify the leading margin computation in isolation rather than bundling it as part of the HtmlParserTest?

Reply:
@BenHenning I have to create a test case in FAQSingleActiviyTest right?

Reply:
@BenHenning I created a new Test suite (CustomBulletSpanTest) similar to HtmlParserTest, Please check

Reply:
Looks good.

Reply:
@FareesHussain I think you should remove this test from here as this needs to be in `CustomBulletSpanTest` as per Ben's comment.

Reply:
@rt4914 I think this test is required in the `HtmlParserTest` too and ben said to update this test case.
Should I remove it?

Reply:
Its fine we can keep this test.